renderdoc/common/dds_readwrite.h

```diff
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "api/replay/data_types.h"
+#include <cstdio>
```

Comment:
`<stdio.h>` instead of `<cstdio>` please.

Reply:
done